.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore this test file
+test.md
```

Comment:
Great job adding  `test.md` file and `node_modules` directory to .gitignore file :heavy_check_mark: